demisto_sdk/commands/format/update_playbook.py

```diff
@@ -76,6 +76,7 @@ def run_format(self) -> int:
             self.update_tests()
             self.update_conf_json('playbook')
             self.add_description()
+            self.delete_sourceplaybookid()
```

Comment:
please add test for that

Reply:
There is already test for that.
check in demisto_sdk/commands/format/tests/test_formatting_yml_test.py

def test_playbook_sourceplaybookid(source_path):
    schema_path = os.path.normpath(
        os.path.join(__file__, "..", "..", "..", "common", "schemas", '{}.yml'.format('playbook')))
    base_yml = PlaybookYMLFormat(source_path, path=schema_path)
    base_yml.delete_sourceplaybookid()

    assert 'sourceplaybookid' not in base_yml.data

Reply:
so how come it did not fail until now ?